style.css

```
@@ -1568,3 +1568,64 @@ body.color-picker-open #renderCanvas {
+  line-height: 1.5em;
+}
+
+#shortcuts-table th {
+  text-align: left;
+  padding-top: 1em;
+  font-size: 1.2em;
+}
+
+kbd {
+  background-color: var(--color-bg);
+  border: 1px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)